charts/postgreslet/templates/configmap-sidecars.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  postgres-exporter-image: {{ .Values.sidecars.exporter.image | quote }}
+  postgres-exporter-container-port: {{ .Values.sidecars.exporter.containerPort | quote }}
+  postgres-exporter-requests-cpu: {{ .Values.sidecars.exporter.resources.requests.cpu | quote }}
+  postgres-exporter-requests-memory: {{ .Values.sidecars.exporter.resources.requests.memory | quote }}
+  postgres-exporter-limits-cpu: {{ .Values.sidecars.exporter.resources.limits.cpu | quote }}
+  postgres-exporter-limits-memory: {{ .Values.sidecars.exporter.resources.limits.memory | quote }}
+  postgres-fluentbit-image: {{ .Values.sidecars.fluentbit.image | quote }}
+  postgres-fluentbit-requests-cpu: {{ .Values.sidecars.fluentbit.resources.requests.cpu | quote }}
+  postgres-fluentbit-requests-memory: {{ .Values.sidecars.fluentbit.resources.requests.memory | quote }}
+  postgres-fluentbit-limits-cpu: {{ .Values.sidecars.fluentbit.resources.limits.cpu | quote }}
+  postgres-fluentbit-limits-memory: {{ .Values.sidecars.fluentbit.resources.limits.memory | quote }}
+  queries.yaml: {{ b64enc .Values.sidecars.exporter.queries }}
+  fluent-bit.conf: {{ b64enc .Values.sidecars.fluentbit.conf }}
+  
+kind: ConfigMap
+metadata:
+  name: {{ include "postgreslet.fullname" . }}-postgres-sidecars
+{{- with .Values.podAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+{{- end }}
+  labels:
+    {{- include "postgreslet.labels" . | nindent 4 }}
+
+{{- /*
+  queries.yaml: |+{{ .Values.sidecars.exporter.queries
+ */}}
```

Comment:
Do we need this `queries.yaml` in `metadata`? Is it possible?

Reply:
Could you please clarify that question?

Reply:
OK, no I get what you mean :-)

You are of course right, we do not need this. And this only works becaus it's a multiline go template comment `{{- /*` `*/}}`.

Reply:
Meaning it isn't even being rendered